docs/usage/create.rst.inc

```diff
@@ -75,7 +75,7 @@ borg create
     +-------------------------------------------------------+---------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------+
     |                                                       | ``-x``, ``--one-file-system``                     | stay in the same file system and do not store mount points of other file systems.  This might behave different from your expectations, see the docs. |
     +-------------------------------------------------------+---------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------+
-    |                                                       | ``--numeric-owner``                               | only store numeric user and group identifiers                                                                                                        |
+    |  (Note: this option is deprecated)                    | ``--numeric-owner`` (Use ``--numeric-id``)        | only store numeric user and group identifiers                                                                                                        |
```

Comment:
first line of this file:

IMPORTANT: this file is auto-generated from borg's built-in help, do not edit!

also, do not regenerate this file to keep the amount of changes to review smaller - when a release is made, all these files will be regenerated/updated.